tests/unit/utils/test_win_reg.py

```diff
@@ -637,6 +637,10 @@ def test_cast_vdata_reg_dword(self):
         result = win_reg.cast_vdata(vdata=vdata, vtype='REG_DWORD')
         self.assertTrue(isinstance(result, six.integer_types))
 
+        vdata = '0000001'
+        result = win_reg.cast_vdata(vdata=vdata, vtype='REG_DWORD')
+        self.assertTrue(isinstance(result, six.integer_types))
```

Comment:
Should the expected value be `1`? If so, this test would be more useful like this:

```suggestion
        vdata = '0000001'
        expected_result = 1
        result = win_reg.cast_vdata(vdata=vdata, vtype='REG_DWORD')
        self.assertEqual(result, expected_result)
```
Rewritten this way the test already checks that the types are correct but *also* that the value gets correctly converted.

Reply:
Damn... I'll make the change. Why you gotta be so smart...?